entity-framework/dotnet-data/index.yml

```diff
@@ -1,80 +1,81 @@
 ### YamlMime:Hub
 
-title: .NET data documentation
-summary: .NET is the open source and cross-platform solution for integrating data, data services, and data backends with your applications.
+title: .NET (C#) data documentation hub
+summary: This page contains resources about how to work with data using .NET (C#). Use our docs and tutorials to choose from our wide variety of available databases & data access approaches and get started quick!
 brand: dotnet
 
 metadata:
-  title: .NET data documentation
-  description: Welcome to the .NET data hub! What data-related task would you like to perform? This page is your gateway to documentation and tutorials for the various ways you interact with data from .NET. 
+  title:  .NET (C#) data documentation hub
```

Comment:
Does it add anything to specifically state "hub" here?

Reply:
Not really, will take off!